function/services/application-mapper/application-question.js

```diff
@@ -122,9 +124,44 @@ function mapApplicationQuestion(data, applicationForm, addressDetails) {
 
             // Check if the applicant is eligible for special expenses
             case data.theme === 'special-expenses':
-                columnValue = applicationForm?.applicant_expenses ? 'true' : 'false';
+                columnValue = applicationForm?.applicant_expenses ? 'true' : data.value;
+                break;
+
+            // Check if the applicant was estranged from the deceased
+            case data.id === 'q-applicant-living-together':
+            case data.id === 'q-applicant-living-apart':
+                columnValue = data.value ? 'N' : 'Y';
+                break;
```

Comment:
Is this logic correct? Those questions seem to be the inverse of each other, it seems strange the answer would be the same regardless? I.E should this column be falsy if `living_together === true` OR `living apart === true`?

Reply:
I thought it was strange too, but the data dictionary reads the following:
![image](https://github.com/CriminalInjuriesCompensationAuthority/tempus-broker/assets/97952806/beccfee3-ae23-4e44-8520-7d7145796804)

Speaking with Julie if `q-applicant-living-together` is true then they won't ever see `q-applicant-living-apart`